ios/FluentUI.Demo/FluentUI.Demo/Demos/CommandBarDemoController.swift

```diff
@@ -75,6 +75,24 @@ class CommandBarDemoController: DemoController {
             }
         }
 
+        var accentImage: UIImage? {
+            switch self {
+            case .delete:
+                return UIImage(named: "delete24Filled")
+            default:
+                return nil
+            }
+        }
+
+        var accentImageTintColor: UIColor? {
+            switch self {
+            case .delete:
+                return Colors.communicationBlue
+            default:
```

Comment:
we try to avoid default and spell out all the enum because default tends to make compiler happy and we can easily make mistakes while adding new enums.

Reply:
Update here as well as other places where default is used in this file.